### PR TITLE
Shell one-liner

### PR DESCRIPTION
**Pull request by timkelty**
Along with https://github.com/craftcms/server-check/pull/16, this will give us a one liner that  can be run to perform the server-check:

```
curl -Lsf https://raw.githubusercontent.com/craftcms/server-check/HEAD/check.sh | bash
```

Part of this is a GH action that creates a release on tag, and adds a tarball asset to the release. I also added the sha256 checksum, but aren't yet using them in the script.

### CODE REVIEW

**Review by angrybrad — COMMENTED** [review]
On the chance that people would want to execute check.sh directly (outside of our use-case), we should document that it won't work on Windows unless you run it under WSL.

File: README.md [hunk 1]
```diff
@@ -2,5 +2,13 @@
...
-You can upload the `server` folder to your web server's public html folder and load `checkit.php` from your browser
+## Usage
+
+Run the following in a terminal of any [\*nix](https://en.wikipedia.org/wiki/Unix-like) environment (e.g. Linux, MacOS, WSL).
+
+```bash
+curl -Lsf https://raw.githubusercontent.com/craftcms/server-check/HEAD/check.sh | bash
+```
+
+Alternatively, you can upload the `server` folder to your web server's public html folder and load `checkit.php` from your browser
```
Comment (timkelty) [inline README.md:13]: @angrybrad hows that sound?

**Review by jasonmccallister — APPROVED** [review]
💯

File: README.md (hunk 1 above)
Comment (angrybrad) [reply README.md:13]: :chefkiss:

**Pull request merged**